examples/instantsearch/instantsearch.html

```diff
@@ -6,17 +6,17 @@
     <script>
       !(function (e, a, t, n, s, i, c) {
         (e.AlgoliaAnalyticsObject = s),
-          (e.aa =
```

Comment:
This isn't also incorrect in the current install snippet is it?

Reply:
yes, this change is unrelated, but the snippet [here](https://github.com/algolia/search-insights.js#1-load-the-library) is correct